Src/ILGPU/IR/Analyses/LoopInfo.cs

```diff
@@ -234,21 +234,36 @@ out InlineList<(PhiValue, Value)> phiValues)
                     outside,
                     inside,
                     branch));
-                phiInductionsVariables.Add(phiValue);
+                visitedPhis.Add(phiValue);
             }
 
-            // Check all
+            // Check all phi values
             foreach (var phi in phis)
             {
                 // Check whether this phi is an induction variable
```

Comment:
QUESTION: Is this comment about `Check whether this phi is an induction variable` still valid?

Reply:
Nice one 👍 Fixed invalid comment 🤓